tests/jwt_vp.rs

```diff
@@ -0,0 +1,117 @@
+use std::str::FromStr;
+use std::sync::Arc;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use anyhow::Result;
+
+use oid4vp::verifier::request_signer::{P256Signer, RequestSigner};
+use ssi_claims::jwt::VerifiablePresentation;
+use ssi_claims::vc::v2::syntax::VERIFIABLE_PRESENTATION_TYPE;
+use ssi_claims::{JWSPayload, JWTClaims};
+// use ssi_claims::vc::v1::VerifiableCredential;
+use ssi_claims::jwt;
+use ssi_dids::ssi_json_ld::syntax::{Object, Value};
+use ssi_dids::ssi_json_ld::CREDENTIALS_V1_CONTEXT;
+use ssi_dids::DIDKey;
+use ssi_jwk::JWK;
+
+#[tokio::test]
```

Comment:
Determine if this test should be ignored so it does not overwrite existing vp token.

NOTE: the vp token expires after 1 hour, so any test checking expiration will need to generate a valid token.